packages/core/core/src/utils/fetch.ts

```diff
@@ -20,9 +20,7 @@ export const createStrapiFetch = (
       ...options,
     };
 
-    if (logs) {
-      strapi.log.debug(`Making request for ${url}`);
-    }
+    if (logs) strapi.log.debug(`Making request for ${url}`);
```

Comment:
We tend to avoid single-line ifs in https://github.com/strapi/strapi :eyes: 
```suggestion 
    if (logs) {
      strapi.log.debug(`Making request for ${url}`);
    }
```